axelor-account/src/main/resources/i18n/messages_fr.csv

```diff
@@ -804,7 +804,7 @@
 "At least a record is linked to this analytic configuration, are you sure you want to proceed ?","Il existe au moins un enregistrement en base en lien avec la configuration analytique, souhaitez-vous continuer ?",,
 "At least one stock move is in litigation, do you really want to validate passed for payment ?","Au moins un mouvement de stock a été rejeté. Êtes-vous certain de vouloir procéder à la validation du BAP ?",,
 "At least one tax accounting move line associated to this move is not correctly set in terms of vat system settings. In order to avoid troubles in the tax declaration, we advise you to use the button generate tax while entering manually a move to avoid such misconfiguration. The vat system value can be changed on the tax move line in the tab 'Others'","Au moins une des lignes d'écriture comptable de taxe associée à cette écriture n'est pas correctement configurée pour le régime de TVA. Pour éviter tout ennui dans la déclaration de taxes, nous vous conseillons plutôt d'utiliser le bouton générer les taxes lorsque vous saisissez des écritures comportant des taxes. Le champ régime de TVA peut-être modifié sur la ligne d'écriture de taxe dans l'onglet Autre",,
-"Attachments",,,
+"Attachments","Fichiers attachés",,

```

Comment:
AOP ?

Reply:
This is a view field listing out the Files. I retook the same entry as in other modules.